book/src/02_core_ledger/00.md

```diff
@@ -1,4 +1,4 @@
-# Jörmungandr User Guide
+# Core Ledger
 
 Welcome to the Jörmungandr User Guide.
```

Comment:
if we're removing references to jorm, this one should probably go

Reply:
It should, I am just waiting on a decision about the name we should use...